src/register-org/components/check-your-answers/check-your-answers.component.ts

```diff
@@ -58,11 +58,12 @@ export class CheckYourAnswersComponent extends RegisterComponent implements OnIn
       this.registerOrgService.postRegistration().subscribe(() => {
         this.router.navigate([this.registerOrgService.REGISTER_ORG_NEW_ROUTE, 'registration-submitted']);
       },
-      (() => {
-        this.validationErrors.push({
-          id: 'confirm-terms-and-conditions',
-          message: this.apiErrorMessage
-        });
+      ((errorResponse) => {
+        const returnedError = { id: 'confirm-terms-and-conditions', message: this.apiErrorMessage };
+        if (errorResponse && errorResponse.status === 400 && errorResponse.error?.errorDescription) {
```

Comment:
This can be written as `if (errorResponse?.status === 400 && errorResponse.error?.errorDescription)`